AI/src/test/java/Test.java

```diff
@@ -0,0 +1,194 @@
+import dk.gruppeseks.bodtrd.ai.Node;
+import dk.gruppeseks.bodtrd.common.data.entityelements.Position;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Stack;
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+/**
+ *
+ * @author Frederik
+ */
+public class Test
```

Comment:
you should clean up this class. its a mess. also why is it that you have copied the astar algorithm to this test class? you should be able to use it by importing instead